site-book/README.md

```diff
@@ -0,0 +1,50 @@
+# Metron Site-Book documentation
+
+Metron's Site Book is an attempt at producing documentation that is
+
+- Versioned and reviewed
+- Tied to code versions
+- Highly local to the code being documented
+
+The idea is that a release manager would build the site-book (following the instructions below), then publish it from the public [Metron site](http://metron.incubator.apache.org/) as the docs for the new released version. Older site-book versions should remain available for users that need them.
```

Comment:
Do we actually have a process for the release manager to follow that includes this?  It should probably exist [here](https://cwiki.apache.org/confluence/display/METRON/Release+Process).  If I'm not missing it, I will happily suggest an update to the wiki via the dev list.